### PR TITLE
Eval script

### PR DESCRIPTION
**Pull request by mfuntowicz**
Added evaluation script to go with the leaderboard

### CODE REVIEW

File: ai_challenge/pig_chase/README.md [hunk 1]
```diff
@@ -66,6 +66,34 @@ python pig_chase_baseline.py -t random
 
 For additional command line options, see the usage instructions: `python pig_chase_baseline.py -h`.
 
+## Evaluate your agent
+
+We provide a commodity evaluator PigChaseEvaluator, which allows you to quickly evaluate
+performances of your agent.
```
Comment (malmo-admin) [inline ai_challenge/pig_chase/README.md]: evaluate performances -> evaluate the performance

Comment (katja-hofmann) [inline ai_challenge/pig_chase/README.md]: evaluate performances -> evaluate the performance

File: ai_challenge/pig_chase/evaluation.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            filepath = abspath(filepath)
+            parent = join(pardir, filepath)
+            if not exists(parent):
+                makedirs(parent)
+
+            with open(filepath, 'w') as f_out:
+                dump(metrics, f_out)
+
+            print('==================================')
+            print('Evaluation done, results wrote at %s' % filepath)
```
Comment (katja-hofmann) [inline ai_challenge/pig_chase/evaluation.py]: results wrote -> results written

**Review by katja-hofmann — APPROVED** [review]
just small typos, otherwise ready to go

**Pull request merged**